core/lib/page-functions.js

```diff
@@ -507,9 +513,7 @@ function getNodeDetails(element) {
   let lhId = window.__lighthouseNodesDontTouchOrAllVarianceGoesAway.get(element);
   if (!lhId) {
     lhId = [
-      window.__lighthouseExecutionContextId !== undefined ?
-        window.__lighthouseExecutionContextId :
-        'page',
+      window.__lighthouseExecutionContextUniqueIdentifier ?? '?',
```

Comment:
`'page'` wasn't super descriptive but it was something to denote the difference between isolated context elements and page-context. We could pick something else